scripts/vet-proto.sh

```diff
@@ -20,18 +20,11 @@ trap cleanup EXIT
 # consistent with the place where all binaries installed by scripts in this repo
 # go.)
 if [[ "$1" = "-install" ]]; then
-  if [[ "${GITHUB_ACTIONS}" = "true" ]]; then
-    PROTOBUF_VERSION=25.2 # Shows up in pb.go files as v4.22.0
-    PROTOC_FILENAME=protoc-${PROTOBUF_VERSION}-linux-x86_64.zip
-    pushd /home/runner/go
-      wget https://github.com/google/protobuf/releases/download/v${PROTOBUF_VERSION}/${PROTOC_FILENAME}
-      unzip ${PROTOC_FILENAME}
-      protoc --version # Check that the binary works.
-    popd
-  else 
-    # TODO: replace with install protoc when https://github.com/grpc/grpc-go/pull/7064 is merged.
-    die "-install currently intended for use in CI only."
-  fi
+    if [[ "${GITHUB_ACTIONS}" = "true" ]]; then
+      source ./scripts/install_protoc.sh "/home/runner/go"
+    else
+      die "run protoc installer https://github.com/grpc/grpc-go/scripts/install_protoc.sh."
```

Comment:
broken link?

Reply:
also shouldn't we mention how to run the installer from the terminal? 

We should also put something in the top level package comment in `scripts/install_protoc.sh`

Reply:
also i'm not convinced that only Github Actions should be able to run `./scripts/vet-proto.sh` with `-install`. I would expect that if a user runs this, then we install protoc for them

Reply:
```https://github.com/grpc/grpc-go/scripts/install_protoc.sh.``` install_protoc.sh script is a new file so this link will be accessed only when it is merged.

Reply:
This is how links to files in our repo @ master would look like `https://github.com/grpc/grpc-go/blob/master/admin/admin.go`


Also about [this](https://github.com/grpc/grpc-go/pull/7064#discussion_r1623049270); why not just something like: 

```go
source ./scripts/install_protoc.sh
```